tutorials/scripting/singletons_autoload.rst

```diff
@@ -73,7 +73,8 @@ in top-to-bottom order.
 
 .. image:: img/autoload_example.webp
 
-This means that any node can access a singleton named "PlayerVariables" with:
+If the **Enable** column is checked (which is the default), then the singleton can
+be accessed directly in GDScript, without requiring ``get_node()``:
```

Comment:
If we remove the `get_node()` example, this could be rewritten to not mention the function at all.
Otherwise we should probably reverse it and add it as: "If you deactivate the `global variable` checkbox, the node can no longer be accessed by its name. Use the `get_node()` function to access it. + example". As the issue suggests.